[2023-08-24 19:22:23,561] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2023-08-24 19:22:23,573] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: entregable3.extract_and_transform_data scheduled__2023-08-23T00:00:00+00:00 [queued]>
[2023-08-24 19:22:23,574] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-08-24 19:22:23,575] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-08-24 19:22:23,575] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-08-24 19:22:23,592] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): extract_and_transform_data> on 2023-08-23 00:00:00+00:00
[2023-08-24 19:22:23,599] {standard_task_runner.py:52} INFO - Started process 10132 to run task
[2023-08-24 19:22:23,612] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'entregable3', 'extract_and_transform_data', 'scheduled__2023-08-23T00:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/entregable3.py', '--cfg-path', '/tmp/tmpqbk072ur', '--error-file', '/tmp/tmpezsdyx2s']
[2023-08-24 19:22:23,619] {standard_task_runner.py:80} INFO - Job 90: Subtask extract_and_transform_data
[2023-08-24 19:22:23,780] {task_command.py:371} INFO - Running <TaskInstance: entregable3.extract_and_transform_data scheduled__2023-08-23T00:00:00+00:00 [running]> on host eebeded14cb4
[2023-08-24 19:22:23,926] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Yamil
AIRFLOW_CTX_DAG_ID=entregable3
AIRFLOW_CTX_TASK_ID=extract_and_transform_data
AIRFLOW_CTX_EXECUTION_DATE=2023-08-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-23T00:00:00+00:00
[2023-08-24 19:22:32,646] {logging_mixin.py:115} INFO - No hay nuevos datos para cargar.
[2023-08-24 19:22:32,647] {python.py:173} INFO - Done. Returned value was: None
[2023-08-24 19:22:32,662] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=entregable3, task_id=extract_and_transform_data, execution_date=20230823T000000, start_date=20230824T192223, end_date=20230824T192232
[2023-08-24 19:22:32,706] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-24 19:22:32,736] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
